src/game/systems.rs

```diff
@@ -0,0 +1,28 @@
+use bevy::prelude::*;
+
+use crate::game::SimulationState;
+
+pub fn toggle_simulation(
```

Comment:
This file could be called pause instead of systems to indicate what it is about. But it can be changed later.